py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -405,7 +405,7 @@ def aten_ops_to_copy_dtype(
     )
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.clone.default)
+@dynamo_tensorrt_converter(torch.ops.aten.clone.default)  # type: ignore[misc]
```

Comment:
Before this is committed check to see if theres a way to fix this mypy error @gs-olive can you look at the decorator as well?

Reply:
I recently noticed that `mypy` does not accurately show errors for these decorators. I will look more into this.

The decorator is already as strongly-typed as it can be though, as here:
https://github.com/pytorch/TensorRT/blob/585b2b66e50a9900f513b3aa495ae3ed210351d4/py/torch_tensorrt/dynamo/conversion/converter_registry.py#L64-L69